cnchi/installation/install.py

```diff
@@ -774,10 +774,10 @@ def update_pacman_conf():
                     else:
```

Comment:
better but we dont want to have lines longer than 99 characters as that is against Python Coding Standards. That's why I recommended doing it the way that I did above.